internal/handlers/pg/msg_find.go

```diff
@@ -154,3 +169,65 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 
 	return &reply, nil
 }
+
+func getMaxTimeMSParameter(document *types.Document) (int32, error) {
```

Comment:
Should we add some documentation like for the other unexported functions?

Reply:
I agree, it was added